### DISCUSSION TITLE
the url is being sent automatically, how to disable this

### DISCUSSION THREAD

**Discussion by RuanMD** [shlinkio/discussions#144] [Help wanted] [answered]
### Shlink version

image: ghcr.io/shlinkio/shlink:3.7.3-roadrunner

### PHP version

unknown

### How do you serve Shlink

Docker image

### Database engine

PostgreSQL

### Database version

14

### Summary

ORIGINAL URL > THE WAY I WANT IT TO LOOK
https://www.voeazul.com.br/br/pt/home/selecao-voo?c[0].ds=VCP&c[0].std=05/29/2024&c[0].as=FOR&c[1].std=06/01/2024&c[1].as=VCP&p[0].t=ADT&p[0].c=2&p[0].cp=false&f.dl=3&f.dr=3&cc=PTS

URL ENCODE > WHEN I ENTER THE SHORT LINK IT LOOKS LIKE THIS
https://www.voeazul.com.br/br/pt/home/selecao-voo?c%5B0%5D.ds=VCP&c%5B0%5D.std=05%2F29%2F2024&c%5B0%5D.as=FOR&c%5B1%5D.std=06%2F01%2F2024&c%5B1%5D.as=VCP&p%5B0%5D.t=ADT&p%5B0%5D.c=2&p%5B0%5D.cp=false&f.dl=3&f.dr=3&cc=PTS

AND THE WEBSITE HAS AN ERROR, HOW CAN I RESOLVE THIS? GENERATE THE URL WITHOUT IT ENCODE

**Comment by acelaya** [comment, marked as answer]
Those are effectively the same URLs, but the second one is properly URL-encoded.

The error seems to be caused by a bug in the destination page, which is not handling URL-encoded arguments.

**Reply by RuanMD** [reply]
This, but when the URL encodes the site does not understand, then it has to be generating the link to be coding, it is possible
